include/picongpu/particles/manipulators/unary/MaxwellJuettner.def

```diff
@@ -0,0 +1,158 @@
+/* Copyright 2014-2022 Rene Widera, Sergei Bastrakov
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "picongpu/simulation_defines.hpp"
+
+#include "picongpu/particles/manipulators/generic/FreeRng.def"
+#include "picongpu/particles/manipulators/unary/FreeTotalCellOffsetRng.def"
+
+#include <pmacc/math/operation.hpp>
+#include <pmacc/random/distributions/Uniform.hpp>
+
+#include <cstdint>
+
+
+namespace picongpu::particles::manipulators::unary
+{
+    namespace acc
+    {
+        /** Functor to modify particle momentum based on temperature for relativistic case
+         * Use Maxwell Juettner Distribution, implementation based on https://doi.org/10.1063/1.4919383
+         * This functor is for the highly-relativistic case.
+         * In this case the added momentum follows the Maxwell-Juettner distribution.
+         *
+         * @tparam T_ValueFunctor pmacc::math::operation::*, binary functor type to
+         *                        add a new momentum to an old one
+         */
+        /** @tparam T_ValueFunctor pmacc::math::operation::*, binary functor type to
+         *                        add a new momentum to an old one
+         * @{
+         */
+
+        /** Version with fixed temperature given via parameter struct
+         *
+         * @tparam T_ParamClass configuration parameter, follows requirements of param::MaxwellJuettnerCfg
+         */
+        /** USE THIS IN THE SAME WAY AS Temperature.hpp JUST REPLACE Temperature with MaxwellJuettner FOR
+         * ALL FUNCTOR AND TEMPLATE NAMES**/
```

Comment:
both comment blocks claim to be documentation of something following them, I think you intended them to be one block.

Reply:
was fixed by @SergeyKonstantinErmakov